Find a functionally equivalent code:android.util.ArraySet.contains:COMMENT
Method Modifier: public      
Comment:/**
 * Check whether a value exists in the set.
 *
 * @param key The value to search for.
 * @return Returns true if the value exists, else false.
 */

@@ -1,3 +1,3 @@
 {
-    return key == null ? (indexOfNull() >= 0) : (indexOf(key, key.hashCode()) >= 0);
+    return indexOf(key) >= 0;
 }

